server/controllers/apply/applicationsController.ts

```diff
@@ -302,4 +302,46 @@ export default class ApplicationsController {
       })
     }
   }
+
+  cancelForm(): RequestHandler {
```

Comment:
I wasn't sure how best to do these controller functions. I need one to render the form view, and the other one to be the endpoint the form POSTs to 🤔

Reply:
We could leave those methods here or extract them into a new  abandon controller? From there we can continue to use conventional naming such as #new #create methods.

Perhaps similar to how we have the PagesController nested under controllers/apply/applications/pagesController.ts